lib/WP_Auth0_LoginManager.php

```diff
@@ -481,29 +564,36 @@ public function auth0_sso_footer( $previous_html ) {
 		}
 	}
 
-	public function auth0_singlelogout_footer( $previous_html ) {
-
-		echo $previous_html;
-
-		if ( ! is_user_logged_in() ) {
-			return;
-		}
-
-		$singlelogout = $this->a0_options->get( 'singlelogout' );
-
-		if ( ! $singlelogout ) {
-			return;
+	/**
+	 * Outputs JS on all pages to log a user out if no Auth0 session is found.
+	 * Hooked to `wp_footer` action.
+	 *
+	 * @see WP_Auth0_LoginManager::init()
+	 */
+	public function auth0_singlelogout_footer() {
```

Comment:
you have changed the signature here, sneaky. This looks like an internal method though so fine.

Reply:
It's hooked to the `wp_footer` action, which does not pass any arguments to that function so that wasn't doing anything anyways. I was talking about this with @lbalmaceda, these callbacks have to be public to be hooked but I can't think of a use case where someone would call this directly, let alone have any use for an already non-functioning parameter.